### PR TITLE
Login

### PR DESCRIPTION
**Pull request by devfajar**
login and register system

### CODE REVIEW

File: app/Controllers/UserController.php [hunk 1]
```diff
@@ -6,130 +6,107 @@
...
-            'email' => 'required|valid_email',
-            'password' => 'required|min_length[5]',
-        ]);
-
-        if (!$inputs) {
-            return view('login', [
-                'validation' => $this->validator,
-            ]);
-        }
+        if ($this->request->getMethod() == 'post') {
```
Comment (bokunodev) [inline app/Controllers/UserController.php:18]: apakah perlu? routing?

Comment (bokunodev) [reply app/Controllers/UserController.php:18]: mending route e di pisah

**Review by bokunodev — CHANGES_REQUESTED** [review]
mensing route login dan validate di pisah

File: app/Controllers/UserController.php (hunk 1 above)
Comment (devfajar) [reply app/Controllers/UserController.php:18]: okay tak pisah e

Comment (devfajar) [reply app/Controllers/UserController.php:18]: sing anyar gak tak routing

**Pull request merged**